ui/src/shared/components/StaticLegend.js

```diff
@@ -0,0 +1,120 @@
+import React, {PropTypes, Component} from 'react'
+import _ from 'lodash'
+import uuid from 'node-uuid'
+
+const style = {
```

Comment:
Please move these styles to the existing `static-legend` rule in `static-legend.scss`